frontend/src/components/RecipeHeader.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components/macro'
+
+export default function RecipeHeader({recipeName, recipeDescription}) {
+    return(
+        <Wrapper>
+            <h3>{recipeName}</h3>
+            <p>{recipeDescription}</p>
+        </Wrapper>
+    )
+}
+
+const Wrapper = styled.section`
```

Comment:
is there a better name for wrapper?  Header for example?

Reply:
switched name